Log total miss inplaced bytes

laithsakka opened the pull request:
Summary: title.

Test Plan: add tests. run existing tests.

Differential Revision: D63411459


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @rec

zou3519 commented on torch/_inductor/fx_passes/reinplace.py:
```diff
@@ -496,23 +496,42 @@ def can_inplace(node, mutated_arg):
+        missed_bytes = 0
+
+        def bytes(node):
+            if node.meta.get("val") is not None:
+                t = node.meta["val"]
+                return t.element_size() * t.numel()
+            else:
+                return 0
```
what does this do when it hits a symint?

laithsakka replied:
the new check is          
   if t is not None and isinstance(t.element_size(), int) and isinstance(t.numel(), int):

laithsakka commented:
address lint errors and export latest

laithsakka committed:
Log total miss inplaced bytes (#136684)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/136684

title.

Test Plan: add tests. run existing tests.

Reviewed By: zou3519

Differential Revision: D63411459
